research/picai/nnunet_scripts/eval.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+
+from picai_eval import evaluate_folder
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--pred_path",
+        type=str,
+        required=True,
+        help="Path to the folder containing the predicted segmentation maps"
+        "Only uses maps whose filename matches a file in the ground truth folder",
+    )
+    parser.add_argument(
+        "--gt_path",
+        type=str,
+        required=True,
+        help="Path to the folder with the ground truth segmentation maps"
+        "Only uses maps whose filename matches a file in the predictions folder",
+    )
+    parser.add_argument(
+        "--output_path",
+        type=str,
+        required=False,
+        default="metrics.json",
+        help="Where to save the metrics as a json. Eg. 'path/to/metrics.json'",
+    )
+
+    args = parser.parse_args()
+
+    # Compute picai metrics
+    metrics = evaluate_folder(
+        y_det_dir=args.pred_path,
+        y_true_dir=args.gt_path,
+    )
+
+    # Print metrics
```

Comment:
Maybe worth mentioning in the comments here or in the README what the metrics being computed are. If its a lot of info you can give a short overview and link to the picai documentation or somewhere else that expands on the metrics